### ISSUE TITLE
@use cannot be used for the same module within different modules

### ISSUE THREAD

**Issue by MrJohz** [connorskees/grass#44]
**Minimal Reproducible Example**:
Consider the three files, `main.scss` (the root module), `utils.scss` (which provides some utility variables/mixins), and `other.scss` (which also uses the utilities):
```scss
// main.scss
@use "utils";
@use "other";
.from-main { color: utils.$value; }

// utils.scss
$value: #fff;

// other.scss
@use "utils";
.from-other { color: utils.$value; }
```

**`grass` Output**:
```scss
Error: There's already a module with namespace "utils".
  ╷
1 │ @use "utils";
  │       ^^^^^^
  ╵
./other:1:7


```

**`dart-sass` Output**:
```scss
.from-other {
  color: #fff;
}

.from-main {
  color: #fff;
}
```

From what I can tell, in `dart-sass`, each module has its own scope for `@use` imports, but `grass` does not.  As a user, the `dart-sass` implementation is what I would expect.

**Comment by connorskees** [comment]
Hello, thank you for such a nice test case. I have resolved this in https://github.com/connorskees/grass/commit/d101a36f0ccf8bbd286fcbf13cff9b08e3d25363 and added two tests. Sorry for letting this sit for so long; I was unable to work on this library for quite some time. I've not decided how exactly I want to release this (minor or bugfix), but I will ping this issue when this fix is released to crates.io.

**Comment by connorskees** [comment]
Fix released in `0.10.7`
